vivisect/tools/graphutil.py

```diff
@@ -629,7 +629,16 @@ def clearGraphRouting(graph):
     for nid, ninfo in graph.getNodes():
         graph.getNodeProps(nid)['up'  ] = False
         graph.getNodeProps(nid)['down'] = False
-    
+
+def reduceGraph(graph, props=['up','down']):
+    '''
```

Comment:
you should never, ever, ever, ever initialize default func/method variables with mutable types... :)